cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/PullCommand.java

```diff
@@ -81,28 +81,34 @@ public void execute() {
 
         String resourceName = argList.get(0);
         String orgName;
+        String packageName;
         String moduleName;
         String version;
 
         // Get org-name
-        int orgNameIndex = resourceName.indexOf("/");
-        if (orgNameIndex != -1) {
-            orgName = resourceName.substring(0, orgNameIndex);
+        String[] module = resourceName.split("/");
+        if (module.length == 2) {
+            orgName = module[0];
+            packageName = module[1];
             if (orgName.equals("ballerina")) {
                 throw LauncherUtils.createLauncherException("`Ballerina` is the builtin organization and its modules"
                                                                     + " are included in the runtime.");
             }
         } else {
-            throw LauncherUtils.createLauncherException("no module-name provided");
+            CommandUtil.printError(outStream,
+                    "Pull command requires the names of the organization and module.",
```

Comment:
```suggestion
                    "pull command requires the names of the organization and module.",
```
Aren't we using simple letters only for warnings?